### ISSUE TITLE
Add PyTorch/Tensorflow interface

### ISSUE THREAD

**Issue by jcreinhold** [jcreinhold/intensity-normalization#28]
## 🚀 Feature
It might be desirable to have some of the normalization schemes available as data augmentation schemes. Add this interface for the relevant schemes.

**Comment by jcreinhold** [comment]
Not going to bother with this. Most normalization methods, besides simple ones like Z-Score, are too computationally intensive to do on the fly. It makes more sense to preprocess the images with normalization and then use the normalized images as input to a network.

Someone else can add this support if they want to take this on—I'll provide guidance.
